mobile/src/actions/index.js

```diff
@@ -1 +1,44 @@
+import { Cozy, MemoryStorage } from 'cozy-client-js'
+
 export const SETUP = 'SETUP'
+export const SET_URL = 'SET_URL'
+
+export const setupServerUrl = (router, location) => {
+  return async (dispatch, getState) => {
+    let cozy = new Cozy({
+      cozyURL: `http://${getState().mobile.serverUrl}`,
+      oauth: {
```

Comment:
The `oauth` object should not be defined for non OAuth client, so maybe the code should more be structured as follow:

```js
let oauth
if (window.cordova && window.cordova.InAppBrowser) {
  oauth = { /* ... */ }
}
let cozy = new Cozy({
  cozyURL: "...",
  oauth: oauth
})
```